<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiPrjWsWidget</class>
 <widget class="QWidget" name="MultiPrjWsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>620</height>
   </rect>
  </property>
    <property name="windowTitle">
     <string>MultiPrjWsWidget</string>
    </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblMPW">
     <property name="text">
      <string>Projects</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="lstProjects"/>
   </item>
  <item row="2" column="0">
   <layout class="QHBoxLayout" name="hboxActions">
    <item>
    <widget class="QPushButton" name="btnAddProject">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    </item>
    <item>
    <widget class="QPushButton" name="btnRemoveProject">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    </item>
   </layout>
  </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
